skimage/transform/hough_transform.py

```diff
@@ -17,15 +17,15 @@ def hough_line(img, theta=None):
         Input image with nonzero values representing edges.
     theta : 1D ndarray of double
         Angles at which to compute the transform, in radians.
-        Defaults to -pi/2 .. pi/2
+        Defaults to -pi/2 .. pi/2 with 180 values.
 
     Returns
     -------
-    H : 2-D ndarray of uint64
+    hspace : 2-D ndarray of uint64
```

Comment:
@JDWarner I agree with you that distances is better but I hesitated to change the API of the hough peak function. I can change that way depending on how the discussion ends.